frappe/website/website_generator.py

```diff
@@ -27,7 +27,10 @@ def get_website_properties(self, key=None, default=None):
 
 	def autoname(self):
 		if not self.name and self.meta.autoname != "hash":
-			self.name = self.scrubbed_title()
+			if self.meta.autoname == "naming_series:":
```

Comment:
@kamaljohnson 

autoname set in the controller has a higher priority while setting names

Adding this custom logic to the actual doctype controller will make more sense